### PR TITLE
Added Vanilla player input

### PR DESCRIPTION
**Pull request by Waterpicker**

### CODE REVIEW

**Comment by Waterpicker** [comment]
Explain please?

File: src/main/java/org/spout/vanilla/component/entity/player/VanillaCamera.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ * more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License,
+ * the MIT license and the Spout License Version 1 along with this program.
+ * If not, see <http://www.gnu.org/licenses/> for the GNU Lesser General Public
+ * License and see <http://spout.in/licensev1> for the full license, including
+ * the MIT license.
+ */
+/*
+ * To change this template, choose Tools | Templates
```
Comment (kitskub) [inline src/main/java/org/spout/vanilla/component/entity/player/VanillaCamera.java]: This needs to be removed.

**Comment by lukespragg** [comment]
This @Waterpicker:

```
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
```

https://github.com/Waterpicker/Vanilla/commit/0891aaf51079d9eeb90299aa15cfd9375e5d1bc9#L1R28

File: src/main/java/org/spout/vanilla/component/entity/player/VanillaCamera.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	@Override
+	public ViewFrustum getFrustum() {
+		return frustum;
+	}
+
+	@Override
+	public Matrix getRotation(){
+		return MatrixMath.createRotated(getOwner().get(Head.class).getRotation());
+	}
+}
```
Comment (windy1) [inline src/main/java/org/spout/vanilla/component/entity/player/VanillaCamera.java]: Missing a new line here at the end of the file.

**Comment by Waterpicker** [comment]
Fixed

**Comment by lukespragg** [comment]
Now you need to squash the commits, keeping the original description. ;)

**Comment by Waterpicker** [comment]
0.o. Oh god. I suck at this part

**Pull request closed without merging**